scripts/wasmd/manual_start.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Have you ever used this script? If not, let's delete it here at least for the new chain.